android/quest/src/main/java/org/smartregister/fhircore/quest/ui/profile/ProfileViewModel.kt

```diff
@@ -262,6 +263,7 @@ constructor(
               )
               .toBoolean()
           }
+          ?.toList()
```

Comment:
Is this toList() necessary for the map operation below? if that is the case then I do not think it is of best interest to use a sequence here.